client/src/components/RatingsComponents/ReviewsList/ReviewsList.jsx

```diff
@@ -4,17 +4,18 @@ import ReviewTile from './ReviewTile.jsx';
 import ReviewsListSorting from './ReviewsListSorting.jsx';
 import NewReviewModal from './NewReviewModal.jsx';
 
-function ReviewsList ({reviews, factors, productName, characteristics, product_Id}) {
+function ReviewsList ({reviews, factors, productName, characteristics, product_Id, handleChangeSort, starsClicked, data}) {
   const [showModal, setShowModal] = useState(false);
   const openFormModal = () => {
     setShowModal(true);
   };
 
   return (
     <div className='reviews-list'>
-      <ReviewsListSorting reviewsNum={reviews.count}/>
+      <ReviewsListSorting reviewsNum={reviews.length}
+        handleChangeSort={handleChangeSort}/>
       <div className='reviews-list-tiles'>
-        {reviews?.results?.map((review) => {
+        {reviews?.map((review) => {
```

Comment:
My ESLint is throwing a tiny fit about this line. If you aren't seeing an issue with it on your end, no worries. Parsing Error, mine says.

![image](https://user-images.githubusercontent.com/85661360/147869061-876f1514-3a80-4de7-8a10-73b1f3fda332.png)

Reply:
eslint doesnt like '.' chaining in this fashion. you can read more about it 
[https://eslint.org/docs/rules/no-unsafe-optional-chaining#rule-details](url)